patches/chrome-browser-app_controller_mac.mm.patch

```diff
@@ -11,30 +11,15 @@ index cdc779979f779bfea2778b492313c028f0f5cab3..109b928101c1a8feb079021bdaa1ff78
      NSApp.delegate = sharedController;
      return sharedController;
    }();
-@@ -859,6 +859,22 @@ class AppControllerNativeThemeObserver : public ui::NativeThemeObserver {
+@@ -850,6 +850,7 @@ class AppControllerNativeThemeObserver : public ui::NativeThemeObserver {
  
    // Initialize the Profile menu.
    [self initProfileMenu];
-+
-+  // Work around a bug in macOS 14. The application name shown in the menu bar
-+  // should be the CFBundleName, but a bug in the menu rewrite causes the
-+  // CFBundleDisplayName to be swapped in. What's happening here is that the
-+  // NSMenuBarDisplayManager registers for a notification that the app name
-+  // changed, and then, when the notification comes in, it calls a Process
-+  // Manager API that returns the CFBundleDisplayName which is not appropriate
-+  // for this use. Work around this by unregistering the NSMenuBarDisplayManager
-+  // for the notification. See https://crbug.com/1487224 and FB13192263.
-+  if (id sharedMenuBarDisplayManager = [NSClassFromString(
-+          @"NSMenuBarDisplayManager") performSelector:@selector(shared)]) {
-+    [NSWorkspace.sharedWorkspace.notificationCenter
-+        removeObserver:sharedMenuBarDisplayManager
-+                  name:@"NSWorkspaceApplicationNameDidChangeNotification"
-+                object:nil];
-+  }
++  BRAVE_APP_CONTROLLER_MAC_MAIN_MENU_CREATED
```

Comment:
why aren't we doing this in BraveAppController::mainMenuCreated instead?

Reply:
cc: @simonhong

Reply:
In short, we could delete `BRAVE_APP_CONTROLLER_MAC_MAIN_MENU_CREATED` patch now.

Background:
This code is for fixing [app name at menubar on macOS 14`(https://github.com/brave/brave-browser/issues/33987).  As it's upstream issue, I picked early the upstream's workaround fix https://chromium-review.googlesource.com/c/chromium/src/+/4908511 before it's merged.
According to https://issues.chromium.org/issues/40934018#comment13, this bug was fixed in macOS 14.2.
So, it's not merged to upstream finally.

Reply:
Removed the patch in https://github.com/brave/brave-core/pull/26537/commits/72a8a206656a3ee121b5474c1734f0a2aa43bfd5